projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -6,220 +6,285 @@ import { Placeholder, Button, Spinner } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 import { useState, RawHTML, useEffect } from '@wordpress/element';
 import { sprintf, __ } from '@wordpress/i18n';
+import { name as aiParagraphBlockName } from './index';
 
-function formatEditorCanvasContentForCompletion( editor ) {
-	const index = editor.getBlockInsertionPoint().index - 1;
-	const allBlocksBefore = editor.getBlocks().slice( 0, index );
-	if ( ! allBlocksBefore.length ) {
-		return '';
-	}
-	return allBlocksBefore
-		.filter( function ( block ) {
-			return block && block.attributes && block.attributes.content;
-		} )
-		.map( function ( block ) {
-			return block.attributes.content.replaceAll( '<br>', '\n\n' );
-		} )
-		.join( '\n\n' );
-}
+const numberOfCharactersNeeded = 36;
```

Comment:
Should we make this all caps and add comments?

Reply:
Done